### PR TITLE
Add route for inspecting sidekiq jobs

### PR DESCRIPTION
**Pull request by AI-Mozi**
#8

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -770,4 +770,19 @@ class App < Sinatra::Base
...
+  get '/queue' do
+    @workers = Sidekiq::Workers.new.map do |_p, _t, worker|
+      payload = JSON.parse(worker["payload"])
+      {
+        queue: worker["queue"],
+        class: payload["class"],
+        args: payload["args"],
+        created_at: Time.at(payload["created_at"]),
+        enqueued_at: Time.at(payload["enqueued_at"]),
+        run_at: Time.at(worker["run_at"])
```
Comment (postmodern) [inline app.rb:782]: Vertically align the values of the Hash.

File: app.rb [hunk 2]
```diff
@@ -770,4 +770,19 @@ class App < Sinatra::Base
     erb :about
   end
 
+  get '/queue' do
+    @workers = Sidekiq::Workers.new.map do |_p, _t, worker|
+      payload = JSON.parse(worker["payload"])
+      {
```
Comment (postmodern) [inline app.rb:776]: Add a newline between 774 and 775.

File: config/sidekiq.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require 'sidekiq'
+require 'sidekiq/api'
```
Comment (postmodern) [inline config/sidekiq.rb:3]: Move this require into `app.rb`, since that's the only play we use the `Sidekiq::API`.

**Review by postmodern — CHANGES_REQUESTED** [review]
Minor style changes.

**Comment by postmodern** [comment]
This is awesome. I'll open separate tickets for some other ideas, like listing currently running jobs at the top, or adding a JavaScript async refresh.

**Comment by AI-Mozi** [comment]
Current design of this view may not be the best you've ever seen, but imo is enough for now.
I've also found out that our dark mode style needs some changes to work well with bulma styles (for example `box` class looks much better on light mode).

File: views/queue.erb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<h1>Currently running jobs</h1>
+
+<% @workers.each do |worker| %>
+  <div class="box">
+    <div>
+      <div>
+        <h2 class="my-0"><%= worker[:class] %></h2>
+        <p class="mb-0"><b>queue:</b> <%= worker[:queue] %></p>
+        
+        <p><b>arguments:</b>
```
Comment (postmodern) [inline views/queue.erb:10]: Use `<strong>` or `<h5>`/`<h6>`.

File: views/queue.erb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+          <% worker[:args].each do |arg| %>
+            <% arg.each do |k, v| %>
```
Comment (postmodern) [inline views/queue.erb:12]: Use `key` and `value`. I'm not a fan of single letter variable names.

File: views/queue.erb [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+              <%= k %>: <%= v %>
```
Comment (postmodern) [inline views/queue.erb:13]: Wrap the key name in `<strong>`.

Comment (postmodern) [inline views/queue.erb:13]: Maybe wrap the value in `<kbd>`?

File: views/queue.erb [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+            <% end %>
+          <% end %>
+        </p>
+        
+        <hr>
+        <div class="columns">
+          <div class="column is-one-third py-0"><b>Created at:</b> <%= worker[:created_at] %></div>
+          <div class="column is-one-third py-0"><b>Enqueued at:</b> <%= worker[:enqueued_at] %></div>
+          <div class="column is-one-third py-0"><b>Run at:</b> <%= worker[:run_at] %></div>
```
Comment (postmodern) [inline views/queue.erb:22]: Use `<strong>` instead of `<b>`.

File: views/queue.erb [hunk 8]
```diff
@@ -0,0 +1,27 @@
+<h1>Currently running jobs</h1>
+
+<% @workers.each do |worker| %>
+  <div class="box">
+    <div>
+      <div>
+        <h2 class="my-0"><%= worker[:class] %></h2>
+        <p class="mb-0"><b>queue:</b> <%= worker[:queue] %></p>
```
Comment (postmodern) [inline views/queue.erb:8]: Use `<strong>` or `<h3>`.

File: app.rb [hunk 9]
```diff
@@ -770,4 +771,19 @@ class App < Sinatra::Base
     erb :about
   end
 
+  get '/queue' do
+    @workers = Sidekiq::Workers.new.map do |_p, _t, worker|
```
Comment (postmodern) [inline app.rb:775]: Use `_process_id` and `_thread_id` so it's obvious what those arguments are for, even though we currently are not using them.

**Review by postmodern — CHANGES_REQUESTED** [review]
Noticed a few things about block variable names and using `<b>` instead of `<strong>`.

**Review by postmodern — APPROVED** [review]

**Pull request merged**